src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -325,15 +337,14 @@ void Terminal::SetBoxSelection(const bool isEnabled) noexcept
 
 // Method Description:
 // - clear selection data and disable rendering it
+#pragma warning(disable : 26440) // changing this to noexcept would require a change to ConHost's selection model
 void Terminal::ClearSelection()
 {
     _selectionActive = false;
     _allowSingleCharSelection = false;
     _selectionAnchor = { 0, 0 };
     _endSelectionPosition = { 0, 0 };
     _selectionVerticalOffset = 0;
-
-    _buffer->GetRenderTarget().TriggerSelection();

```

Comment:
hmm? we don't need to do this?

Reply:
Decided to move it over to TermControl. That way we ensure that we're only calling `TriggerSelection` once.